articles/flows/guides/device-auth/includes/request-device-code.md

```diff
@@ -0,0 +1,73 @@
+## Request Device Code
+
```

Comment:
This looks like a repeat of `articles/api/authentication/api-authz/_device-code.md`. The other article has much more code samples it seems ,but is lacking an explanation of the response.

Why do we have this duplicate content? Is there a chance we can unify this?

Reply:
This is an include file that serves as part of a tutorial. The other doc you mention is the API reference, which mimics the Swagger-generated Mgmt API ref. 

For every flow, we include the API reference as well as a microsite (for entry-level users), a flow (with more mid-level detail), and a tutorial (with suggestions to use our quickstarts/SDKs, where available, and a basic "roll-your-own" implementation). For the moment, we don't intend to unify these items as they serve different purposes for their intended audiences.

Reply:
As we improve our API refs, though, I hope we can bring these more closely in line and/or unify this info.

Reply:
For reference, here is the tutorial:
https://docs-content-staging-pr-7461.herokuapp.com/docs/flows/guides/device-auth/call-api-device-auth

Here is the API ref:
https://docs-content-staging-pr-7461.herokuapp.com/docs/api/authentication#device-authorization-flow